extensions/components/com_redcore/site/views/translation/tmpl/modal-edit.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
```

Comment:
this file seems similar if not the same as extensions/components/com_redcore/admin/views/translation/tmpl/modal-edit.php
maybe put content of those 2 files into another layout and load them both, so you got only one file